nano/node/write_database_queue.cpp

```diff
@@ -15,8 +15,8 @@ nano::write_guard::~write_guard ()
 	cv.notify_all ();
 }
 
-nano::write_database_queue::write_database_queue () :
-// clang-format off
+nano::write_database_queue::write_database_queue ()
+	: // clang-format off
 guard_finish_callback ([&queue = queue, &mutex = mutex]() {
```

Comment:
Again placement of comment looks strange

Reply:
I don't think that can be controlled, but in this case the format directive can enclose the entire constructor, which is empty.

Reply:
Updated